src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -561,14 +560,14 @@ function getLabelsConfiguration(chartOptions: IChartOptions, _config: any, chart
 
     const style = getLabelStyle(type, stacking);
 
-    const drilldown =
-        stacking || isTreemap(type)
-            ? {
-                  activeDataLabelStyle: {
-                      color: "#ffffff",
-                  },
-              }
-            : {};
+    // const drilldown =
```

Comment:
If we are sure we don't want the code here, please delete it completely (in case we want it back, we can look it up in git)